_pages/phone-screens.md

```diff
@@ -15,70 +15,114 @@ subnav:
 
-2. **Quickly screen out candidates who obviously won't work out.** Typically this is true for candidates who don't share our values, as that's something that's hard to assess from a resume but can shake out very quickly in a conversation. Occasionally it's about technical fit: a candidate has a resume that passed review, but in talking to them it's clear they want to work on things that don't really match what we do (or the resume made it sound like they did work that they seem to not really have done).
-
-Phone screens are performed by someone on the 18F Engineering Hiring team, and often the same person performs all phone screens for a given hiring group.
+2. **Screen out candidates who would not be successful.** Candidates who don't share our values, do not match the required skills for the position, or are interested in working on things or in ways which we cannot offer.
 
-Most candidates pass the phone screen. It's a rough filter: it can't be used to really qualify a candidate, but it can be used to disqualify. It's also not uncommon for candidates to self-select out after the phone screen. This is particularly true for candidates who are financially-motivated; the GS-scale can be below what some candidates feel is appropriate.
+Phone screens are a rough filter: it can't be used to really qualify a candidate, but it can be used to disqualify.
 
-**Because phone screens rely heavily on personal judgment, and are inherently less scientific than the full suite of interviews, phone screeners should be especially wary about unconscious bias.**
+**Because phone screens are decided upon by a single individual, phone screeners should be especially wary about their unconscious bias.**
 
 Please remember to take notes and pass them along: they can be very useful to future interviewers.
 
 ## Before the call
 
 The candidate will have already received guidance from TTS Talent on what it’s like to interview at TTS and an explainer of payscales and term limits. This guide changes depending on the hiring authority used and the specifics of the role, but here’s an example [candidate guidance for a DHA role](https://docs.google.com/document/d/10n9DnWP07fjsRcM0Yw3z6OlbWi2nHcg0cKUhXIy-3po/edit) and for [a reserved hire role](https://docs.google.com/document/d/1ci_PgaJty6esQ_C0yk8DKP_8ySpdaxR9plVbK7AO95o/edit).
 
-Take a few minutes to familiarize yourself with the candidate’s resume and review [what to look for during a phone screen](https://docs.google.com/document/d/1La5M7YojZFEc0lvpQZ4Kc7dL8UgssBoJe3FqmmJwmK4/edit).
+Take a few minutes to familiarize yourself with the candidate’s resume, copy the notes template, and review what to look for during a phone screen.
```

Comment:
Do we still have a separate phone screen google doc? Might be handy to link to it here.

Reply:
it's currently linked in the button just below. Should I clarify that those are the same thing?